docs/INSTALL.md

```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
 $ gcloud auth application-default login
 ```
 
-### Custom terraform-dcos variables
+#### Supported Operating Systems
+
+Here is the [list of operating systems supported](https://github.com/dcos-terraform/terraform-template-gcp-tested-oses/tree/master/platform/cloud/gcp).
+
+#### Supported DC/OS Versions
+
+Here is the list of DC/OS versions supported on dcos-terraform natively:
 
-The default variables are tracked in the [variables.tf](../variables.tf) file. Since this file can be overwritten during updates when you may run `terraform get --update` when you want to fetch new releases of DC/OS to upgrade too, its best to use the cluster_profile.tfvars and set your custom terraform and DC/OS flags there. This way you can keep track of a single file that you can use manage the lifecycle of your cluster.
+- [OSS Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/open/dcos-versions)
+- [Enterprise Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/ee/dcos-versions)
 
-To apply the configuration file, you can use this command below.
+**Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>Master DC/OS version is not meant for production use. It is only for CI/CD testing.</p>

Reply:
Remove content from line #61 and copy-paste the above line.